src/main/java/com/programmers/springbootbasic/mediator/requestGenerator/RegisterUserVoucherGenerator.java

```diff
@@ -0,0 +1,28 @@
+package com.programmers.springbootbasic.mediator.requestGenerator;
+
+import com.programmers.springbootbasic.domain.userVoucherWallet.presentation.dto.CreateUserVoucherWalletRequest;
+import com.programmers.springbootbasic.infrastructure.IO.ConsoleInteractionAggregator;
+import com.programmers.springbootbasic.mediator.ConsoleRequest;
+import com.programmers.springbootbasic.presentation.MainMenu;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RegisterUserVoucherGenerator implements MenuRequestGenerator {
```

Comment:
```suggestion
public class RegisterUserVoucherGenerator implements MenuRequestGenerator<CreateUserVoucherWalletRequest> {
```

ConsoleResult에 제네릭타입으로 CreateUserVoucherWalletRequest가 정의되어 있는 것 처럼, 
MenuRequestGenerator<CreateUserVoucherWalletRequest> 로 선언하여 타입을 명시해주는 것 어떠신가요?